src/wp-admin/includes/privacy-tools.php

```diff
@@ -400,6 +400,17 @@ function wp_privacy_generate_personal_data_export_file( $request_id ) {
 
 	// Convert the groups to JSON format.
 	$groups_json = wp_json_encode( $groups );
+	if ( false === $groups_json ) {
+		$groups_json = '"false"';
```

Comment:
Change to `'"null"'` for consistency as a value here adds no value nor is descriptive to indicate the cause that generated it.

Reply:
Resolved in commit https://github.com/WordPress/wordpress-develop/pull/1115/commits/8c6f37f1c2f845e485befa07668bd95ccc4d10c1